onnxruntime/hosting/main.cc

```diff
@@ -1,15 +1,73 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#include <iostream>
+#include <string>
+#include "beast_http.h"
+#include "boost/program_options.hpp"
 #include "core/session/inference_session.h"
 
+namespace po = boost::program_options;
+
+void test_request(const std::string& name, const std::string& version,
+                  const std::string& action, Http_Context& context) {
+  std::stringstream ss;
+
+  ss << "\tModel Name: " << name << std::endl;
+  ss << "\tModel Version: " << version << std::endl;
+  ss << "\tAction: " << action << std::endl;
+  ss << "\tHTTP method: " << context.request.method() << std::endl;
+
+  http::response<http::string_body>
+          res{std::piecewise_construct, std::make_tuple(ss.str()), std::make_tuple(http::status::ok, context.request.version())};
+
+  res.set(http::field::server, BOOST_BEAST_VERSION_STRING);
+  res.set(http::field::content_type, "plain/text");
+  res.keep_alive(context.request.keep_alive());
+  context.response = res;
+}
+
 int main(int argc, char* argv[]) {
-  using namespace onnxruntime;
+  std::string model_path;
```

Comment:
I think it is ok to keep local variables, but consider create a configuration class to manage configurations. A TODO is ok to me here.

Reply:
I've added a TODO